### ISSUE TITLE
EOTW Addition

### ISSUE THREAD

**Issue by TMNetwork** [drtshock/HCFProblems#72]
So there is something called end of the world on all HCF servers, basically everything become unclaimed and all factions become -100.00 DTR.

Warzone/Safezone don't become unclaimed.

./f eotw toggle - perm node (only ops)

**Comment by MoeXIV** [comment]
It's usually not setting everyones DTR to -100. 
It disables claiming in the world, and unclaims everyones claims. It doesn't matter if they have full DTR or not because they can't claim land.

No "f home"s because there are no claims, :+1:

**Comment by TMNetwork** [comment]
No, there is f homes on EOTW. Just nothing is protected @MoeXIV

**Comment by nsporillo** [comment]
Setting this to help wanted so people can chime in about how this new feature would be implemented.

I will not be adding a new feature that has conflicting specifications, if there is a clash between ideas then we can make something configurable.

**Comment by MoeXIV** [comment]
Not on Kohi or HCT...

**Comment by TMNetwork** [comment]
@MoeXIV Yes. I have people who played both of there maps that said that is how they have it. Check yourself.

**Comment by MoeXIV** [comment]
You "have" people who have. I HAVE played every Kohi EOTW since Map 12. Most or all are on my channel. The whole map is claimed as Warzone, all claims are overridden by it. 
There is meant to be no "f home" in EoTW. It's meant to be hardcore for a reason. It's in the name "HCF".

So before you tell me to "Check myself", please provide evident information on your topic, or I will.

**Comment by TMNetwork** [comment]
@MoeXIV check urself.

**Comment by Doxify** [comment]
@TMNetwork You should really be more respectful.

As for EOTW there shouldn't be any f homes, we don't even need anything added to the factions plugin to do an EOTW. What my staff were planning on doing was completely disabling the factions plugin.

**Comment by TMNetwork** [comment]
@Doxify on ParaPvP, they have /f home and I liked the idea because of bunkers, so it configurable then.

**Comment by TMNetwork** [comment]
@Doxify I hope you know @MoeXIV is my friend and he knows im kidding.

**Comment by LOOOOOOOL** [comment]
@nsporillo can we get this going, my map ends in 1 week and we need to do an EOTW.
Just make it if ur op or have a perm node (f.eotw)
then you do /f start EOTW and basically it will make all faction's territories raid able  (not unclaim, so they can still /f home).


Simple as that. Make warzone and safezone immune to this please. thanks bro.

**Comment by nsporillo** [comment]
@LOOOOOOOL if we add features, we want to make sure they're designed so they can be used by all servers in their own way. With that said, it's going to take time and I can't promise any delivery date. Keep in the mind that valuable time spent replying to issues like https://github.com/drtshock/HCFProblems/issues/130 , https://github.com/drtshock/HCFProblems/issues/126, etc waste time that could be spent working on a new feature.

**Comment by Doxify** [comment]
This command is not need at ALL.

It can be simply done by editing a few lines in the configs :)

**Comment by MoeXIV** [comment]
That is true, but EOTW is a giant part of HCF and adding a little kick for servers using this plugin may be pretty cool. It also saves time for those in the which lack the the common sense to edit those few lines of configuration. In my case (which can be done fairly easy in game) would be to unclaim all non warzone claims, and then disabling claiming in said world. Simple as pie, eh?

**Comment by LOOOOOOOL** [comment]
@nsporillo dude, my map ends in 1 week and we need this stuff done:
results: http://strawpoll.me/4172449/r

**Comment by MoeXIV** [comment]
Of course players want it, it's easier on them. If they have the option to do so, then yes, they are going to say yes. It makes the last few hour of the map more fun, and a better experience. I really feel f home shouldn't be added 👎🏾

**Comment by LOOOOOOOL** [comment]
Just make it frwakjn configurable. Jdk what else nick needs at this point
with all this info....

**Comment by drtshock** [comment]
@LOOOOOOOL if you want features implemented on a deadline, I'd be happy to draw up a client contract for you to pay Puzl's $40/hr rate so we can prioritize you with our other clients. Otherwise, we get to things when we have time :)

**Comment by LOOOOOOOL** [comment]
Will I get that part of the plugin for myself for the $40?

**Comment by drtshock** [comment]
No as we don't like maintaining separate branches for everyone. Huge headache when we have to merge things.

**Comment by LOOOOOOOL** [comment]
Ok well hope to see this soon then.

**Comment by nsporillo** [comment]
There is no such thing as setting DTR to -100 (the min is defined in config, that is not overridden under any circumstance)

I suppose a way to do this is to set every factions dtr to the min in config and setting them as DTR frozen (with an artificially high freeze time). Thus all normal factions become raidable but they can still use /f home or whatever-This wont be configurable (why should it be, adding more configuration for something so small makes things unnecessarily confusing). If a server doesnt like that feature then dont use the /f eotw, just use another means of ending the world (disabling the plugin?)

(for my future reference)
1. Set all factions dtr to min (makes them raidable)
2. Set all factions lastDeath to now
3. Set hcf.dtr.dtr-freeze to like 24 hours or something
4. No faction can regain dtr because frozen, so they stay raidable.

**Comment by LOOOOOOOL** [comment]
What is the lastDeath now thing?
And can u do /f eotw start/stop so when we stop it, the plugin can function
normally?

**Comment by nsporillo** [comment]
It has not been added yet, those were just for my future reference when I go to add this...

lastDeath is just a variable every faction has, is set whenever someone dies to the current time.

**Comment by LOOOOOOOL** [comment]
ok

**Comment by MoeXIV** [comment]
My mistake.

**Comment by TemptingMC** [comment]
@drtshock no longer needed, I had my developers code a bypass for this. Close this issue.

**Comment by drtshock** [comment]
@TemptingMC we'd appreciate it if you PRd your additions upstream. But just because you have it to yourself does not mean that this issue should be closed.

**Comment by TemptingMC** [comment]
@drtshock yea ik, but I was the guy who originally posted this; I was gonna ask for it to be closed since no one else said that wanted it in (in the comments atleast)

**Comment by SaiCoFlu** [comment]
I'd like to see it @dtrshock , keep it open, thanks.

**Comment by Lotteries** [comment]
I'd love to see this aswell! @drtshock  and @nsporillo Keep it open!

**Comment by KriticalPvP** [comment]
+1 I'd like it

**Comment by TemptingMC** [comment]
Weird that everyone wants it all of a sudden when I got terrible feedback about this command when I needed it.. ;/

Seems a bit unfair, but idc anymore lol.

**Comment by KriticalPvP** [comment]
Well, I think It's because a lot of servers are having their own EOTW's atm. such as HydraPvP.

I'd still like it, EOTW is soon for me :)

**Comment by drtshock** [comment]
Keep discussion relevant to the ISSUE (@TemptingMC). I do not care if someone already had someone do it for them privately.

**Comment by nsporillo** [comment]
The original spec was for a /f eotw start and /f eotw end. I dont really think it's something that you can just turn back off. We'd have to keep a copy of all factions prior eotw so we can revert back if needed and it just doesnt seem like a good idea to do that.

Instead, /f eotw would irreversibly end the world.. so just be an OP to do this.

**Comment by TemptingMC** [comment]
Not needed anymore, lol. This was opened 2 months ago.

**Comment by SaiCoFlu** [comment]
@TemptingMC No one cares what you need or don't need, this plugin is made for other people, not only you.

**Comment by TemptingMC** [comment]
@flu I originally was the one who made this post dumbass.

**Comment by HeyImYellow** [comment]
Bump. We probably need this more than any of the other stuff that's on hold, it's so annoying to run the EOTW...

**Comment by TemptingMC** [comment]
Yea, when I needed it, they didn't rush it. I'd just wait as they don't
rush anything (atleast if i ask).

**Comment by drtshock** [comment]
Sorry (but not actually) that we develop on our own schedules and not yours.

**Comment by TemptingMC** [comment]
3 months man ;/ rly?
It's ok though.

**Comment by drtshock** [comment]
It doesn't help that the reporter has been banned off of the issue tracker multiple times.

**Comment by TemptingMC** [comment]
I was banned 1 time and it was 2 months in before I was banned. I also was
banned for stating my opinion.

**Comment by TemptingMC** [comment]
It doesn't matter anymore. This is irrelevant since my devs added it in for
me.

**Comment by SaiCoFlu** [comment]
Not trying to offend you, but you'd do us all a favour by leaving already, your opinions are not needed and neither is your constant repetition of "my dev done it for me" bullshit. G'day.
